client/js/screens/EventBooking/EventBookingContainer.js

```diff
@@ -0,0 +1,101 @@
+import React, {Component} from 'react';
+import {TouchableOpacity, Image, StatusBar} from 'react-native';
+import {withCollapsible} from 'react-navigation-collapsible';
+import {THEME} from '../../config';
+import EventBooking from './EventBooking';
+
+class EventBookingContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      headerHeight: 88,
+    };
+  }
+
+  static navigationOptions = ({navigation}) => {
+    const themeColor = navigation.getParam('themeColor') || 'light';
+
+    return {
+      title: 'Event Booking',
+      headerBackTitle: true,
+      headerTitleStyle: {
+        color:
+          themeColor === 'light'
+            ? THEME.colors.astronautBlue
+            : THEME.colors.white,
+      },
+      headerTransparent: true,
+      headerStyle: {
+        backgroundColor: 'transparent',
+      },
+
+      headerBackground: () => {
+        return (
+          <StatusBar
+            barStyle={themeColor === 'light' ? 'dark-content' : 'light-content'}
+          />
+        );
+      },
+
+      headerLeft: () => {
+        return (
+          <TouchableOpacity
+            style={{marginLeft: 12}}
```

Comment:
Move this inline style from here to styles.js file

Reply:
Changed